src/services/baseApi.ts

```diff
@@ -6,18 +6,23 @@ type baseApiProps = {
     authInstance: IAuthService;
```

Comment:
As the baseApi isn't used anymore, it could probably be removed. It mingt still be in use in the testing context, but that should also be changed into working the same way as the actual procosysApi does now.